src/views/Home/components/Menu/index.js

```diff
@@ -0,0 +1,280 @@
+import React, { Component } from 'react';
```

Comment:
What is this file here for?  It seems to duplicate much of ../DiningBalance/index.js, and it's important to minimize duplicate code.